### ISSUE TITLE
Constant crashes

### ISSUE THREAD

**Issue by Deeringdipper** [frostwire/frostwire#981]
Used to be once and awhile or maybe a few times a day it crashed now it's constantly every 10/15 minutes to as little time as getting through 1 song then it crashes think by now after years you'd fix the crashes love the app but the crashing is getting old

**Comment by gubatron** [comment]
can you be more specific as to what app you're talking about?
Are you in windows, macos, linux, android?
What version of the app are you running?
What operating system are you running?

**Comment by Deeringdipper** [comment]
It's v.2.4.3 on Android not sure on anything else I've always had crash
problems but never to this extent  and I've been using the app for years

**Comment by gubatron** [comment]
Thank you, 2.4.5 is out and we're about to release 2.4.6. Many crashes have been fixed since 2.4.3, please stay alert for our next update 2.4.6 (could be as soon as this afternoon, working on one last bug) a lot is in the pipeline since 2.4.3, here's the changelog since that version you're runing, lots of fixes:

```
FrostWire 2.4.6 build 689 - JAN/24/2022
 - jlibtorrent 1.2.15.2
 - compileSdkVersion 31
 - com.google.android.material:material:1.5.0
 - androidx.browser:browser:1.4.0
 - com.google.firebase:firebase-crashlytics-gradle:2.8.1
 - com.google.firebase:firebase-bom:29.0.4
 - androidx.appcompat:appcompat:1.4.1
 - androidx.core:core-ktx:1.7.0
 - androidx.media2:media2-session:1.2.0
 - androidx.media2:media2-widget:1.2.0
 - androidx.media2:media2-player:1.2.0
 - com.google.android.gms:play-services-ads:20.4.0
 - com.applovin:applovin-sdk:10.3.5
 - com.mopub.mediation:applovin:10.3.4.0
 - com.android.tools.build:gradle:7.1.0
 - androidx.preference:preference:1.2.0
 - Chinese translation update by TheJokerW

FrostWire 2.4.5 build 688 - JAN/07/2022
 - jlibtorrent 1.2.15.1 (boost 1.78, openssl 1.1.1m)
 - lt:don't disable announcing from local endpoints because of temporary failures
 - lt:fix issue in parsing UPnP XML response with multiple forwarding services
 - lt:cache DNS lookups for SOCKS5 proxy
 - lt:fix stalled pieces on disk-full errors
 - lt:fix build configuration issue on NetBSD, OpenBSD and DragonFly
 - lt:make UTF-8 sanitization a bit stricter. This will re-write invalid UTF-8 code points encoding surrogate pairs
 - lt:fix restoring last_seen_complete from resume data
 - lt:fix issue on MacOS where the DHT was not restarted on a network-up notification
 - lt:make remove_torrent flags be treated as flags (instead of an enum)

FrostWire 2.4.4 build 687 - NOV/23/2021
 - jlibtorrent 1.2.14.2 (rebuild with boost 1.77)
 - smaller installer size
 - androidx.annotation:annotation:1.3.0
 - com.google.firebase:firebase-crashlytics-gradle:2.8.0

FrostWire 2.4.3 build 686 - OCT/22/2021
 - Fixes search bug that would occur after exiting and reopening app
 - Better search http connection management
 - androidx.media2:media2-*:1.2.0 update
 - targetSdkVersion 31 (Android 12)
 - com.applovin:applovin-sdk:10.3.4
 - com.mopub.mediation:applovin:10.3.3.0
 - com.android.tools.build:gradle:7.0.3

FrostWire 2.4.2 build 684 - SEP/21/2021
 - Search screen rewritten, considerably faster search results
 - WRITE_EXTERNAL and WRITE_SETTINGS permissions no longer required
 - Android 11+ compatibility updates in the way the app interacts with the Android Media Store API
 - After Android 10, files can no longer be deleted from "My Files", this has been delegated to
   the OS's File Manager, less is more.
 - gradle:4.1.0 -> 4.2.2 -> 7.0.2
 - Java 11 language support, common/ and android/ codebased will be modernized through future updates
 - Storage location selection feature dropped, the application will use external shared storage where available by default
 - minSdkVersion 24 (Android 7.0)
 - targetSdkVersion 30 (Android 11)
 - New jlibtorrent 1.2.14.1 (no more netlink sockets, ifaddr sockets)
 - Set as Ringtone action removed (needed WRITE_SETTINGS) permission
 - Soundcloud search fixed
 - lt:fix issue on MacOS where the DHT was not restarted on a network-up notification
 - lt:make remove_torrent flags be treated as flags (instead of an enum)
 - lt:improve handling of seed flag in PEX messages
 - lt:fix issue of accruing unlimited DHT node candidates when DHT is disabled
 - lt:fix bug in parsing chunked encoding
 - lt:fix incorrect reporting of active_duration when entering graceful-pause
 - com.google.gms:google-services:4.3.10
 - com.google.firebase:firebase-crashlytics-gradle:2.7.1
 - com.google.android.flexbox:flexbox:3.0.0
 - com.mopub:mopub-sdk:5.18.0
 - com.mopub.mediation:unityads:3.7.5.0
 - com.mopub.mediation:applovin:10.3.3.0
 - com.unity3d.ads:unity-ads:3.7.5
 - com.applovin:applovin-sdk:10.3.2
 - com.google.android.material:material:1.4.0
 - androidx.exifinterface:exifinterface:1.3.3
 - androidx.appcompat:appcompat:1.3.1
 - androidx.recyclerview:recyclerview:1.2.1
 - androidx.core:core-ktx:1.6.0
 - com.google.android.gms:play-services-ads:20.2.0
 - com.android.billingclient:billing:4.0.0
 - com.google.android.gms:play-services-ads:20.3.0
 - com.google.code.gson:gson:2.8.8
 - com.google.re2j:re2j:1.6
 - com.squareup.okhttp3:okhttp:5.0.0-alpha.2
 - com.android.tools.build:gradle:7.0.2
 ```

**Comment by gubatron** [comment]
Here's a new release, hopefully it solves your crashes
https://github.com/frostwire/frostwire/releases/download/frostwire-android-2.4.6-build-689/frostwire-android-release-2.4.6-b689-plus.apk
